### PR TITLE
SUS-573 examine usage of Gatherer extension

### PR DESCRIPTION
**Pull request by ghost**
https://wikia-inc.atlassian.net/browse/SUS-573

This extension is only enabled on http://mtg.wikia.com/ and implements a special page http://mtg.wikia.com/wiki/Special:Gatherer

It is a third party and does not seem to be actively developed.

It is supposed to help create and update articles.

If the input is "Zombie Master", we expect some data to be fetched from http://gatherer.wizards.com/Pages/Card/Details.aspx?name=zombie%20master

I have changed the URL base (no longer beta) and added logging. Let's give it a try and consider sunsetting this feature.

@macbre

### CODE REVIEW

File: extensions/3rdparty/Gatherer/Gatherer_body.php [hunk 1]
```diff
@@ -317,6 +320,8 @@ function fetchCardInfo( $url ) {
                                 $text = false;
 			}
 
+			Wikia\Logger\WikiaLogger::instance()->info( __METHOD__, [ 'curl_getinfo' => curl_getinfo( $c ) ] );
```
Comment (macbre) [inline extensions/3rdparty/Gatherer/Gatherer_body.php:323]: Can we log cases when `$this->err` is set as errors?

File: extensions/3rdparty/Gatherer/Gatherer_body.php [hunk 2]
```diff
@@ -439,15 +444,18 @@ private function getSets( $sets ) {
 			if( !array_key_exists( $s, $this->rels ) ) {
 				continue;
 			}
-			$exp = "http://beta.gatherer.wizards.com/Handlers/Image.ashx?type=symbol&set={$this->rels[$s]['abbr']}&size=small&rarity={$r}";
+			$exp = "http://gatherer.wizards.com/Handlers/Image.ashx?type=symbol&set={$this->rels[$s]['abbr']}&size=small&rarity={$r}";
 			$ret[$s][$r] = @file_get_contents( $exp );
```
Comment (macbre) [inline extensions/3rdparty/Gatherer/Gatherer_body.php:448]: Can we make an explicit HTTP call here?

File: extensions/3rdparty/Gatherer/Gatherer_body.php [hunk 3]
```diff
@@ -439,15 +444,18 @@ private function getSets( $sets ) {
...
 			if( !$ret[$s][$r] ) {
 				unset( $ret[$s] );
 			}
 			if( $ret == array() ) {
 				$this->err = 'gatherer-imgerr';
 			}
+
+			Wikia\Logger\WikiaLogger::instance()->info( __METHOD__, [ 'url' => $exp ] );
```
Comment (macbre) [inline extensions/3rdparty/Gatherer/Gatherer_body.php:456]: Log errors - same as above

**Comment by macbre** [comment]
LGTM (apart from me nitpicking above ;))

**Comment by ghost** [comment]
I will refactor the extension and move it to `extensions/wikia` after I know something about its usage. I will then address all issues, such as `@file_get_contents`, overall error handling, tests and everything.

Though, there is a fair chance we will simply retire it as a piece of legacy code enabled on a _single_ wiki.

**Comment by macbre** [comment]
Sure, let's push it live, mistrzu :)

**Pull request merged**